website/src/data/users.tsx

```diff
@@ -1352,6 +1352,14 @@ const Users: User[] = [
     source: 'https://github.com/pingcap/ossinsight',
     tags: ['opensource', 'design'],
   },
+  {
+    title: 'Outerbounds',
+    description: 'A documentation site authored from jupyter notebooks using nbdoc and docusaurus.',
+    preview: require('./showcase/outerbounds.png'),
+    website: 'https://outerbounds.com',
+    source: 'https://github.com/outerbounds/nbdoc',
```

Comment:
Please only add the repo with Docusaurus-related code.

Reply:
Thank you - I've removed the non-Docusaurus repo and set `source: null`.